js/index.js

```diff
@@ -41,3 +41,19 @@ messageForm.addEventListener('submit', (event) => {
     messageList.appendChild(newMessage);
     messageForm.reset();
 });
+var githubRequest = new XMLHttpRequest();
+githubRequest.open('GET', 'https://api.github.com/users/suarez888/repos');
+githubRequest.send();
+githubRequest.addEventListener("load" , function () {
+    let repositories = JSON.parse(this.response);
+    console.log(repositories);
+
+    var projectSection = document.getElementById('projects');
+    var projectList = projectSection.querySelector('ul');
+    for (let i = 0; i < repositories.length; i++ ){
+        var project = document.createElement('li');
+        project.innerText = repositories[i].name;
+        projectList.appendChild(project);
+    };
```

Comment:
You asked: 

> I am having trouble writing a hyperlink in js. My code is:
> ```
> <a href="${repositories[i].url}"> ${repositories[i].name} </a>
> ```
> they display on the page, but the links don't work. 

That's not the code I see here. Is it possible you just haven't saved your latest changes before reloading the page? Or maybe you have multiple copies of the repo, and you're editing a different copy than you have up in the browser? Both of those have happened to me.